common/reviews/api/core.api.md

```diff
@@ -1184,6 +1224,11 @@ interface IGeometryLoadObject {
 
 // @public (undocumented)
 interface IImage {
+    // (undocumented)
+    bufferView?: {
+        buffer: ArrayBuffer;
+        offset?: number;

```

Comment:
Please require the offset if you are using a bufferView at all - that just makes things a bit more consistent, and adding in 0 is pretty easy.